pulp_docker/app/views.py

```diff
@@ -14,3 +57,235 @@ def get(self, request):
         Return a response to the "GET" action.
         """
         return response.Response({})
+
+
+class ServeContentMixin(object):
+    """
+    A mixin that helps serve content using the enabled web server
+    """
+
+    def _django(self, path, headers):
+        """
+        The content web server is Django.
+
+        Stream the bits.
+
+        Args:
+            path (str): The fully qualified path to the file to be served.
+
+        Returns:
+            StreamingHttpResponse: Stream the requested content.
+
+        """
+        try:
+            file = FileWrapper(open(path, 'rb'))
+        except FileNotFoundError:
+            return HttpResponseNotFound()
+        except PermissionError:
+            return HttpResponseForbidden()
+        response = StreamingHttpResponse(file)
+        response['Content-Type'] = headers['Content-Type']
+        response['Docker-Distribution-API-Version'] = 'registry/2.0'
+        response['Content-Length'] = os.path.getsize(path)
+        response['Content-Disposition'] = \
+            'attachment; filename={n}'.format(n=os.path.basename(path))
+        return response
+
+    def _apache(self, path):
+        """
```

Comment:
If this doesn't yet work for apache or nginx, maybe it would be best to remove these for now.